src/Examples/SurrMod/FlowsheetOptimization/Best_practices_optimization.ipynb

```diff
@@ -4636,32 +4648,32 @@
        "</div>"
       ],
       "text/plain": [
-       "                         alamo    pysmo_poly     pysmo_rbf    pysmo_krig  \\\n",
-       "IPOPT iterations            13            13            14            13   \n",
-       "Solve time               0.014         0.009         0.036         0.027   \n",
-       "fs.bypass_frac             0.1           0.1           0.1           0.1   \n",
-       "fs.ng_steam_ratio     1.138445      1.116544      1.124305      1.124276   \n",
-       "fs.steam_flowrate     1.241674      1.217787      1.226153      1.226191   \n",
-       "fs.reformer_duty   39390.78895  38919.350216  39062.177089  39072.795322   \n",
-       "fs.AR                 0.004107      0.004107      0.004107      0.004107   \n",
-       "fs.C2H6               0.000406       0.00053      0.000545      0.000519   \n",
-       "fs.C3H8               0.000089      0.000116      0.000119      0.000114   \n",
-       "fs.C4H10              0.000051      0.000066      0.000068      0.000065   \n",
-       "fs.CH4                0.012764      0.016538      0.016991      0.016224   \n",
-       "fs.CO                 0.104531      0.104842      0.104856      0.104849   \n",
-       "fs.CO2                0.053986      0.053503      0.053528       0.05353   \n",
-       "fs.H2                 0.331559      0.331671      0.331688      0.331698   \n",
-       "fs.H2O                0.151056      0.148642      0.148918      0.148931   \n",
-       "fs.N2                     0.34          0.34          0.34          0.34   \n",
-       "fs.O2                      0.0          -0.0           0.0           0.0   \n",
+       "                          alamo    pysmo_poly     pysmo_rbf    pysmo_krig  \\\n",
+       "IPOPT iterations             19            13            14            13   \n",
```

Comment:
I don't see any updates in the ALAMOPy code since these notebooks were added that would alter the surrogate form, and I'm unaware of any changes to the solver itself.

Reply:
Does the sampling tool box use random sample to separate the data sets? If so, that could be the reason.

This would also be an indicator that for testing we should be seeding the sampler to ensure we get the same result each time.

Reply:
Yes, the data sampling does randomly separate the data for training/validation, that could be the reason.